introduction.md

```diff
@@ -1,3 +1,8 @@
 This is an introduction.
 
-I really like the Ruby course path!
\ No newline at end of file
+I really like the Ruby course path!
+
+Deadly Skills: 
+* Git
+* Ruby
+* C
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?